src/components/avatar-rectangle/styles.ts

```diff
@@ -0,0 +1,77 @@
+import { styled } from '@linaria/react'
+import { ReactComponent as ResidentialPlaceholderIcon } from './icons/residential-placeholder-icon.svg'
+import { ReactComponent as CommercialBottomPlaceholderIcon } from './icons/bottom-commercial-placeholder.svg'
+import { ReactComponent as CommercialSmallBottomPlaceholderIcon } from './icons/bottom-commercial-placeholder-small.svg'
+
+const baseCommercialMediumImagesStyle = `
+  width: 54px;
+  height: var(--size-size-8, 40px);
+  flex-shrink: 0;
+`
+
+const baseCommercialBottomPlaceholderStyle = `
+  position: absolute;
+  bottom: 0;
+  right: 0;
+`
+
+export const ElAvatarRectBottomPlaceholder = styled(CommercialBottomPlaceholderIcon)`
+  ${baseCommercialMediumImagesStyle}
+  ${baseCommercialBottomPlaceholderStyle}
+`
+export const ElAvatarRectBottomPlaceholderSmall = styled(CommercialSmallBottomPlaceholderIcon)`
+  ${baseCommercialMediumImagesStyle}
+  ${baseCommercialBottomPlaceholderStyle}
+`
+
+export const ElAvatarRectCommercialPlaceholder = styled.div`
+  ${baseCommercialMediumImagesStyle}
+  position: absolute;
+  top: 0;
+  left: 0;
+  background: var(--fill-colour-fill-default-light, #e5e9ed);
+`
+
+export const ElAvatarRectangleWrapper = styled.span`
```

Comment:
**suggestion:** I forgot to mention, but I think this styled component should be called `ElAvatarRectangle`. I don't think "Wrapper" is necessary here.